### PR TITLE
[adr 2.0] Use Payload::getSetting for PlatesFormatter

### PR DESCRIPTION
**Pull request by AlexMasterov**
The tests will fall because [adr 2.0.0-alpha2](https://github.com/equip/adr/releases/tag/2.0.0-alpha2) is not available (yet) on [Packagist](https://packagist.org/packages/equip/adr).

**UPD.** The test was updated. Now everything should be fine.

### CODE REVIEW

File: src/Formatter/PlatesFormatter.php [hunk 1]
```diff
@@ -26,27 +26,19 @@ public function __construct(Engine $engine)
...
 
-    /**
-     * @param Template $template
-     * @param PayloadInterface $payload
-     * @return string
-     */
-    protected function render(Template $template, PayloadInterface $payload)
-    {
-        return $template->render($payload->getOutput());
+        return $this->engine->render($template, $output);
```
Comment (shadowhand) [inline src/Formatter/PlatesFormatter.php]: Didn't realize you could do this. Perfect!

**Comment by shadowhand** [comment]
Not sure why the alpha2 tag wasn't available. I pushed it again and now it appears.

**Comment by shadowhand** [comment]
The test failure is because we're not locking to `alpha2`, right?

**Comment by AlexMasterov** [comment]
@shadowhand Yes. But this PR can wait for the stable version of ADR interface.

**Comment by shadowhand** [comment]
@AlexMasterov rebase this to `dev`, we have a stable release for equip/adr now.

**Comment by AlexMasterov** [comment]
@shadowhand done.

**Comment by shadowhand** [comment]
👍

**Pull request merged**